lib/glific/misc/flags.ex

```diff
@@ -198,6 +198,25 @@ defmodule Glific.Flags do
     end
   end
 
+  @doc """
+  Get whatsapp group value for organization flag
+  """
+  @spec get_whatsapp_group_enabled(map()) :: boolean
+  def get_whatsapp_group_enabled(organization) do
+    app_env = Application.get_env(:glific, :environment)
+
+    cond do
+      FunWithFlags.enabled?(:is_whatsapp_group_enabled, for: %{organization_id: organization.id}) ->
+        true
+
+      trusted_env?(app_env, organization.id) ->
+        true
+
+      true ->
+        false
+    end
+  end
+
```

Comment:
this is starting to get a repeated code see if u can optimise this and use single function

Reply:
okay sure